.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "tabWidth": 4,
+  "singleQuote": true,
+  "trailingComma": "all"
```

Comment:
This just aligns with what we currently had and I'd argue that `singleQuote` and `trailingComma` is something we should have part of our template.

Also, should we make `tab` the default?

Reply:
For accessibility, I support the `tab` by default.